manifests/hive_v1alpha1_syncset.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Is this file for testing?

Reply:
Yes this is temporary for testing purposes.

Reply:
Removed testing files.